docs/community/com-config/com-config-home.md

```diff
@@ -3,8 +3,76 @@ title: Community configuration
 layout: default
 parent: Community
 has_children: true
-nav_order: 6
+nav_order: 8
 has_toc: false
 ---
 
-# Community configuration
+FeatureBase features can be enabled or disabled via
+
+* Command-line flags
+* TOML files
+
+## Before you begin
+
+* {% include /com-install/com-install-before-begin.md %}
+
+<!-- waiting on content from com-config-auth branch in PR now
```

Comment:
are we still waiting?

Reply:
Nope. all sorted. fixed :D (and good catch btw)